### ISSUE TITLE
failure completely broken in FormRestResponse

### ISSUE THREAD

**Issue by geoffreydudragne** [denimgroup/threadfix#1148]
It looks like the 2 last commits on ./threadfix-service-interfaces/src/main/java/com/denimgroup/threadfix/webapp/config/FormRestResponse.java  ( c10b26ee97cb38d16c6badba3708e6c32eb2f8b6 and 27fbdaef38d1647f8725d1749e46ca6705875a04 ) completely broke that function:
FormRestResponse<T> failure(String response, BindingResult result)

They change its behaviour and and generate a null pointer exception 100% chances:
basically set errors = null
and call
resultMap.putAll(errors);

So it breaks all the "failure" responses of all the controllers which use that function.
